test/plugin_functional/test_suites/core_plugins/ui_plugins.ts

```diff
@@ -31,22 +32,38 @@ export default function({ getService, getPageObjects }: PluginFunctionalProvider
         await PageObjects.common.navigateToApp('settings');
       });
 
-      it('should attach string to window.corePluginB', async () => {
-        const corePluginB = await browser.execute('return window.corePluginB');
-        expect(corePluginB).to.equal(`Plugin A said: Hello from Plugin A!`);
+      it('should run the new platform plugins', async () => {
+        expect(
+          await browser.execute(() => {
+            const { setup } = ((window as unknown) as CoreProvider).__coreProvider;
+            return setup.plugins.core_plugin_b.sayHi();
```

Comment:
I feel bad about this now that I know that vscode doesn't behave as IDEA regarding `declare` statement. This is indeed less elegant that the previous solution. We should decide with the team if we want to 'allow' these `declare` statements for extensions that are not necessarily accessible from everywhere in the code.

Reply:
ok, I will revert this change